<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>uservklad</class>
 <widget class="QDialog" name="uservklad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>318</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbID">
        <property name="text">
         <string>Вклад</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="leID"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbNazv">
        <property name="text">
         <string>Название</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="leNazv"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbCountry">
        <property name="text">
         <string>Страна</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="leCountry"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lbLoan">
        <property name="text">
         <string>Ставка</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="leLoan"/>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lbTerm">
        <property name="text">
         <string>Срок</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="leTerm"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>428</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnSelectAll">
     <property name="text">
      <string>Выбрать всё</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" rowspan="4" colspan="2">
    <widget class="QTableWidget" name="twVklad"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnExit">
     <property name="text">
      <string>На главную</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Подключение</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>235</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
